src/urbit/lib/chess.hoon

```diff
@@ -345,20 +398,33 @@
         %none       ''
       ==
   --
+::
+::  XX: implement or delete
+::
+::  would read position from fen,
+::  but hasn't been implemented
 ++  fen-to-position
   |=  fen=@t
   ~&  %not-implemented  !!
 --
 |%
+::
+::  miscellaneous game logic
++|  %game-logic
+::
 ++  with-board
   |_  board=chess-board
+::
+::  scan the board for a white king
   ++  white-king
     ^-  chess-square
     ~|  'missing white king'
     =<  p  %-  head
     %+  skim  ~(tap by board)
     |=  [square=chess-square piece=chess-piece]
     =([%white %king] piece)
+::
+::  scan the board for a black king
```

Comment:
Indent both comments two more spaces. Text should probably be "*the* black king".